app/views/users/show.html.erb

```diff
@@ -1,16 +1,30 @@
-<h1><%= current_user.first_name %>'s Folio</h1>
-
-<table>
-  <tr>
-    <td>
-      <%= link_to "Create New Folder", new_folder_path(@current_folder), class: "btn" %>
-    </td>
-    <td>
-      <%= render partial: "shared/upload_form" %>
-    </td>
-  </tr>
-</table>
-<br>
-<%= render partial: "shared/breadcrumbs" %>
-<br>
-<%= render partial: "shared/data_table" %>
+<h1><%= @user.first_name %>'s Folio</h1>
```

Comment:
This file needs to be changed back to the previous version that uses the partials.